src/gui/editorwidgets/qgscheckboxwidgetwrapper.cpp

```diff
@@ -25,21 +25,32 @@ QgsCheckboxWidgetWrapper::QgsCheckboxWidgetWrapper( QgsVectorLayer *layer, int f
 QVariant QgsCheckboxWidgetWrapper::value() const
 {
   QVariant v;
+  if ( config( QStringLiteral( "AllowNullState" ) ).toBool() && mCheckBox && mCheckBox->checkState() == Qt::PartiallyChecked )
+  {
+    return v;
```

Comment:
how do you feel above removing "v" here entirely, and always just directly returning the value? It'd be a bit cleaner IMO vs the current mix of return direct vs set and later return.

Reply:
@nyalldawson , by that you mean only having one return v; at the very end of the function? Sure, no strong feelings either way.

Reply:
I'd lean toward the other way, and just always returning straight away